doc/build_install/windows/Readme.md

```diff
@@ -0,0 +1,312 @@
+## **Building, Using and Debugging OSL in Windows**
+
+#### Copyright Contributors to the Open Shading Language project.
+
+---
+
+### **Contents**
+
+1. Introduction
+   - Experimental
+   - My Setup
+   - Inspiration
+2. Preparing Environment
+   - Cleaning Up the PATH
+   - Set-Up needed Environment Variables
+   - Set Flags based on your needs
+3. Building (Release, Debug)
+   - Running the code based on your config
+   - Be Patient
+   - Errors
+4. Running
+5. Development in Windows
+
+---
+
+### **1. Introduction**
+
+This document is prepared for those who have problem in building OSL in Windows OS, this is completely custom setup. And it is developing to make it compatible with all situations.
+
+(`Disclaimer: This Process is completely Experimental and it worked on my system, maybe you get error with changing some configs, during this process). At a first glance it seems to be complicated, but it is very easy to do, bare with me :)
```

Comment:
I think you have an opening single quote here, that does not have a corresponding closing quote.

Reply:
I fixed this one